test/util/oauthUtil.js

```diff
@@ -186,22 +188,25 @@ define(function(require) {
       authClient.tokenManager.on('refreshed', function() {
         refreshDeferred.resolve();
       });
-      authClient.tokenManager.on('expired', function() {
+      authClient.tokenManager.on('error', function() {
         refreshDeferred.resolve();
       });
       promise = refreshDeferred.promise;
     }
 
     if (opts.fastForwardToTime) {
-      util.warpByTicksToUnixTime(opts.fastForwardToTime);
+      // Since the token is "expired", we're going to attempt to
+      // retrieve it and kick-off the autoRefresh and let the event listeners
+      // above pick up the 'refreshed' and 'error' events.
+      promise = authClient.tokenManager.get(opts.autoRefreshTokenKey);
```

Comment:
@rchild-okta - Following up per offline convo, we're now ensuring the `tokenManager.get` promise is returned and evaluated [below](https://github.com/okta/okta-auth-js/pull/128/files#diff-6563c87f06eb3af6c119bad7f2a64d75R205).

This allows us to verify the following:
  - `tokenManager.get` returns an `OAuthError` when failing to refresh the token
  - `tokenManager.get` returns `undefined` for an expired token
  - `tokenManager.get` returns a refreshed token